dataform/Examples/Dataform_Assertion_Unit_tst/definitions/tests/test_date_assertions.sqlx

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config {
+    type: "test",
+    dataset: "date_assertions"
+}
+
+js {
+    const test_cases = {
+        // Assertions that pass
```

Comment:
We should ideally have better comments than these. The TRUE or FALSE expected values already self describe what these comments say